lib/membrane/rtp/demuxer.ex

```diff
@@ -97,6 +97,14 @@ defmodule Membrane.RTP.Demuxer do
                 This option determines the action to be taken after a stream has been announced with a 
                 `t:new_rtp_stream_notification/0` notification but the corresponding pad has not been connected within the specified timeout period.
                 """
+              ],
+              use_srtp: [
+                spec: false | {true, [ExLibSRTP.Policy.t()]},
```

Comment:
This `{true, policies}` tuple seems weird to me. Maybe the option could be `srtp: false | policies`?